src/jsMain/kotlin/io/beatmaps/playlist/playlist.kt

```diff
@@ -315,15 +315,33 @@ val playlistPage = fc<Props> {
                                 draggable(it.map.id, idx) {
                                     attrs.classes = setOf("drag-beatmap")
 
-                                    beatmapInfo {
-                                        obj = it.map
-                                        version = it.map.publishedVersion()
-                                        this.audio = audio
+                                    removeMapPlaylist {
+                                        attrs.obj = it.map
+                                        attrs.audio = audio
+                                        attrs.playlistKey = playlist.playlistId
+                                        attrs.mapId = it.map.id
+                                        attrs.removeMap = {
+                                            setMaps(maps - it)
+                                        }
                                     }
                                 }
                             }
                         }
                     }
+                } else if (playlist?.owner?.id == userData?.userId){
```

Comment:
Jenkins will be upset with the spacing here

Reply:
Lol I am too, hadn't noticed.